src/devices/cpu/z80/z80.cpp

```diff
@@ -566,13 +462,13 @@ uint16_t z80_device::arg16()
  ***************************************************************/
 inline void z80_device::eax()
 {
-	m_ea = (uint32_t)(uint16_t)(IX + (int8_t)arg());
+	m_ea = (u32)(u16)(IX + (int8_t)arg());
 	WZ = m_ea;
 }
 
 inline void z80_device::eay()
 {
-	m_ea = (uint32_t)(uint16_t)(IY + (int8_t)arg());
+	m_ea = (u32)(u16)(IY + (int8_t)arg());
 	WZ = m_ea;
 }
```

Comment:
You’re mixing the unabbreviated `int8_t` with the abbreviated `u32` and `u16` here now.